docs-src/docs/guides/walkthrough.md

```diff
@@ -0,0 +1,317 @@
+* `--port 80:80/HTTP`: instructs the load balancer to forward requests from port 80 on the server to port 80 in the Docker container using the HTTP protocol
+* `demo-env`: the name of the environment in which you are creating the load balancer
+* `guestbook-lb`: a name for the load balancer itself
+
+---
+
+### Part 3: Deploy the Docker Task Definition
+
+The `deploy` command is used to specify the Docker task definition that refers to a web application.
+
+Here, we'll create a new deploy called **guestbook-dep** that refers to the **Guestbook.Dockerrun.aws.json** file you obtained earlier. At the command prompt, execute the following:
+
+`l0 deploy create Guestbook.Dockerrun.aws.json guestbook-dep`
+
+This will give us the following output:
+
+```
+DEPLOY ID        DEPLOY NAME    VERSION
+guestbook-dep.1  guestbook-dep  1
+```
+
+The following is a summary of the arguments passed in the above command:
+
+* `deploy create`: creates a new deployment and allows you to specify a Docker task definition
+* `Guestbook.Dockerrun.aws.json`: the file name of the Docker task definition (use the full path of the file if it is not in your current working directory)
+* `guestbook-dep`: a name for the deploy, which you will use later when you create the service
+
+_The Deploy Name and Version are combined to create a unique identifier for a deploy. If you create additional deploys named **guestbook-dep**, they will be assigned different version numbers._
+
+
+---
+
+### Part 4: Create the Service
+
+The final stage of the deployment process involves using the `service` command to create a new service and associate it with the environment, load balancer, and deploy that we created in the previous sections. The service will execute the Docker containers which have been described in the deploy.
+
+Here, we'll create a new service called **guestbook-svc**. At the command prompt, execute the following:
+
+`l0 service create --loadbalancer demo-env:guestbook-lb demo-env guestbook-svc guestbook-dep:latest`
+
+This will give us the following output:
+
+```
+SERVICE ID    SERVICE NAME   ENVIRONMENT  LOADBALANCER  DEPLOYMENTS       SCALE
+guestbo9364b  guestbook-svc  demo-env     guestbook-lb  guestbook-dep:1*  0/1
+```
+
+The following is a summary of the arguments passed in the above command:
+
+* `service create`: creates a new service
+* `--loadbalancer demo-env:guestbook-lb`: the fully-qualified name of the load balancer; in this case, the load balancer named **guestbook-lb** in the environment named **demo-env**. 
+	- _(It is not strictly necessary to use the fully qualified name of the load balancer, unless another load balancer with exactly the same name exists in a different environment.)_
+* `demo-env`: the name of the environment you created in Part 1
+* `guestbook-svc`: a name for the service you are creating
+* `guestbook-dep`: the name of the deploy that you created in Part 3
+
+
+---
+
+### Check the Status of the Service
```

Comment:
Often times, you end up needing to use `service logs` to see why your service is or isn't coming up. We may want to add some commentary here to show that's a thing that can help verify the service is working-ish.